src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -731,7 +731,28 @@ public final class SQLServerDriver implements java.sql.Driver {
     static final String AUTH_DLL_NAME = "mssql-jdbc_auth-" + SQLJdbcVersion.MAJOR + "." + SQLJdbcVersion.MINOR + "."
             + SQLJdbcVersion.PATCH + "." + Util.getJVMArchOnWindows() + SQLJdbcVersion.RELEASE_EXT;
     static final String DEFAULT_APP_NAME = "Microsoft JDBC Driver for SQL Server";
+    static final String APP_NAME_TEMPLATE = "Microsoft JDBC - %s, %s - %s";
```

Comment:
should this have "Driver" and "SQL Server" in there? old name is "Microsoft JDBC Driver for SQL Server" new name will just be something like "Microsoft JDBC - Windows 11, 17.0.1+12-39- amd64"